src/OrchardCore.Modules/OrchardCore.Media/Views/Admin/MediaApplication.cshtml

```diff
@@ -128,8 +128,8 @@
 <input type="hidden" id="uploadFiles" value="@Url.Action("Upload", "Admin", new { area = "OrchardCore.Media" })" />
 
 @*Localizable Strings shared*@
-<input type="hidden" id="deleteFolderMessage" value="@T["Are you sure you want to delete this folder?"]" />
-<input type="hidden" id="deleteMediaMessage" value="@T["Are you sure you want to delete this media?"]" />
+<input type="hidden" id="deleteFolder" data-title="@T["Delete media folder"]" data-message="@T["Are you sure you want to delete this folder?"]" />
+<input type="hidden" id="deleteMedia" data-title="@T["Delete media"]" data-message="@T["Are you sure you want to delete this/these media?"]" />
```

Comment:
I like the "data-" technique you use here, its seems more concise to me. The only thing is that in the same file we have two different ways of doing things.  Feel free to keep it like that or change it to have two have different inputs : "deleteFolderTitle" , "deleteFolderMessage" etc...

Reply:
The technique that I am using will reduce the number of the hidden fields are required